.pre-commit-config.yaml

```diff
@@ -0,0 +1,45 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
+    hooks:
+    -   id: check-yaml
```

Comment:
Would also recommend ` -   id: check-json`

Reply:
as far as I know, this project versions no json files. I would add this check-json hook when json files start being versioned.